media-video/obs-studio/metadata.xml

```diff
@@ -17,8 +17,10 @@
     <flag name="luajit">Build support for scripting via Luajit.</flag>
     <flag name="nvenc">Enable NVENC encoding for nVidia video cards.</flag>
     <flag name="python">Build support for scripting via Python 3.</flag>
+    <flag name="rtmps">Build support for RTMPS.</flag>
```

Comment:
This would be a new local use flag when several other packages already use `rtmp` for the same functionality; not everyone might know that RTMPS is, so better spell it out; you could also use the `<pkg>` tag to throw in the info that mbedtls is going to be used.

Reply:
Yeah, I considered explaining it more, but I'm not sure why I ended up not going for it.  I'll look into it.